am-ui/src/assets/i18n/de.json

```diff
@@ -64,5 +64,20 @@
     "no": "Nein",
     "no-details-found": "Es konnte keine Anzeige gefunden werden",
     "availability-agreement": "Auf Anfrage"
+  },
+  "ad": {
+    "title": "Meine Anzeigen",
+    "sort": "Sortieren nach:",
+    "1": "Zimmer",
+    "2": "Wohnung",

```

Comment:
Is it possible to use english word instead of numbers?

Reply:
I had the same idea but with this solution you can directly use the id from the topic. Else you need to add conditions and everything.